### ISSUE TITLE
System.ArgumentOutOfRangeException

### ISSUE THREAD

**Issue by TheFibonacciEffect** [PowerShell/PSReadLine#1498]
I was tinkering arround with python. Everything worked fine, until I wrote the following line:
`print(data[:][0])`
data is a list comprehension consisting out of tuples with `(x,y)` coordinates.
I guess, the expression in itself is not valid in python (even so I wouldnt know another way to try it).

```
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new       
----------------------------------------------------------------------- 
Last 200 Keys:
 t s Ctrl+Alt+\ C o d e Ctrl+Alt+\ P y t h o n Space P r o j e c t s Ctrl+Alt+\ M u s i c Ctrl+Alt+\ P a r t i t u r Ctrl+Alt+\ m u s i c Ctrl+Alt+\ S c r i p t s Ctrl+Alt+\ p y t h o n . e x e ' Space ' c : Ctrl+Alt+\ U s e r s Ctrl+Alt+\ c a s p a Ctrl+Alt+\ . v s c o d e Ctrl+Alt+\ e 
x t e n s i o n s Ctrl+Alt+\ m s - p y t h o n . p y t h o n - 2 0 2 0 . 4 . 7 6 1 8 6 Ctrl+Alt+\ p y t h o n F i l e s Ctrl+Alt+\ l i b Ctrl+Alt+\ p y t h o n Ctrl+Alt+\ d e b u g p y Ctrl+Alt+\ n o _ w h e e l s Ctrl+Alt+\ d e b u g p y Ctrl+Alt+\ l a u n c h e r ' Space ' d e m o . p 
y ' Space Enter


Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.  
Parameter name: left
Actual value was -1.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.Internal.VirtualTerminal.set_CursorLeft(Int32 value)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, 
Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
----------------------------------------------------------------------- 
```

**Comment by TheFibonacciEffect** [comment]
Acctually as of right now, nothing works, even when I remove the line.
It gives the `ArgumentOutOfRangeException` sometimes when I re-run and sometimes it gives
```
At line:1 char:90
+ ... python.exe" "c:/Users/caspa/Documents/Code/Python 
Projects/Music/Part ...
+
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Unexpected token '"c:/Users/caspa/Documents/Code/Python 
Projects/Music/Partitur/demo.py"' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParentContainsError  
   RecordException
    + FullyQualifiedErrorId : UnexpectedToken```

**Comment by TheFibonacciEffect** [comment]
Okay, I think it had nothing to do with that perticular python line, it is a valid expression and after closing and reopening VS code everythin works fine.

**Comment by daxian-dbw** [comment]
This issue was fixed (see #1306). Please upgrade to the 2.0.1 version of PSReadLine from PowerShellGallery. See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions.
